### PR TITLE
Initial drop of FAT for JPA ORM Diagnostics

### PR DESCRIPTION
**Pull request by jgrassel**
Signed-off-by: Joe Grassel <jgrassel@us.ibm.com>

Initial drop of the FAT project for the JPA ORM Diagnostics.  Consists of an update to fattest.simplicity which adds a dump() command to LibertyServer, and a simple test that ensures that the dump was generated, and extracts JPARuntimeInspector.txt from the dump, and performs some basic checks against the content of the JPA dump.

### CODE REVIEW

File: dev/com.ibm.ws.jpa.container.ormdiagnostics_fat/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
```
Comment (jhanders34) [inline dev/com.ibm.ws.jpa.container.ormdiagnostics_fat/bnd.bnd]:
```suggestion
# Copyright (c) 2018 IBM Corporation and others.
```

File: dev/com.ibm.ws.jpa.container.ormdiagnostics_fat/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+javac.source: 1.8
+
+src: \
+	fat/src, \
+	test-applications/jpasimple/src
+
+fat.project: true
+
+# Dependencies may be local bundles (e.g. com.ibm.websphere.javaee.servlet.3.1)
+#      or binaries from Artifactory (e.g. commons-logging:commons-logging)
```
Comment (jhanders34) [inline dev/com.ibm.ws.jpa.container.ormdiagnostics_fat/bnd.bnd]:
```suggestion
#      or binaries from an archive repository like Maven (e.g. commons-logging:commons-logging)
```

File: dev/com.ibm.ws.jpa.container.ormdiagnostics_fat/bnd.bnd [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+-buildpath: \
```
Comment (jhanders34) [inline dev/com.ibm.ws.jpa.container.ormdiagnostics_fat/bnd.bnd]: This list in the build path seems excessive.  Do you really need cdi, ejb, jms and validation?

File: dev/com.ibm.ws.jpa.container.ormdiagnostics_fat/bnd.bnd (hunk 1 above)
Comment (jgrassel) [reply dev/com.ibm.ws.jpa.container.ormdiagnostics_fat/bnd.bnd]: Thanks for catching that, fixing

File: dev/com.ibm.ws.jpa.container.ormdiagnostics_fat/bnd.bnd (hunk 2 above)
Comment (jgrassel) [reply dev/com.ibm.ws.jpa.container.ormdiagnostics_fat/bnd.bnd]: ditto

File: dev/com.ibm.ws.jpa.container.ormdiagnostics_fat/bnd.bnd (hunk 3 above)
Comment (jgrassel) [reply dev/com.ibm.ws.jpa.container.ormdiagnostics_fat/bnd.bnd]: For the simple JPA app that I use, nah, I don't need all those.

Comment (jgrassel) [reply dev/com.ibm.ws.jpa.container.ormdiagnostics_fat/bnd.bnd]: Removed, and pushed an update commit including all of your recommended changes.

**Pull request merged**